resotolib/resotolib/config.py

```diff
@@ -143,6 +146,10 @@ def load_config(self, reload: bool = False) -> None:
                 log.debug("Starting config event listener")
                 self._ce.start()
 
+    def add_config_load_hook(self, hook: Callable[[Json], Json]) -> None:
+        """Add a hook to be called after the config is loaded."""
+        self._config_load_hooks.append(hook)
+
```

Comment:
When would it make sense to have a configuration that does not replace env vars?
Maybe we should simply do this here automatically?